python-project-template/.initialize_new_project.sh

```diff
@@ -44,6 +44,7 @@ python -m pip install -e . > /dev/null
 
 echo "Installing developer dependencies in local environment"
 python -m pip install -e .'[dev]' > /dev/null
+if [ -f docs/requirements.txt ]; then pip install -r docs/requirements.txt; fi
```

Comment:
Please use `python -m pip`
```suggestion
if [ -f docs/requirements.txt ]; then python -m pip install -r docs/requirements.txt; fi
```